### ISSUE TITLE
"component list" and changing the language do not work in the profile

### ISSUE THREAD

**Issue by tomkolp** [WeblateOrg/weblate#5653]
**Describe the issue**
It is not possible to change the interface language in the profile. Selecting "component list" as the default dasboard view does not work.

**Screenshots**

![image](https://user-images.githubusercontent.com/39633242/111066419-318ee380-84bf-11eb-9c6c-7618e10c66a7.png)
![image](https://user-images.githubusercontent.com/39633242/111066459-7b77c980-84bf-11eb-92a3-5af2225c488c.png)

**Additional context**

docker bleeding image (14.03.2021)

**Comment by tomkolp** [comment]
1 I showed all the lists on the dashboard
2 I selected one list in the profile
3 I turned off the display of all lists on the dashboard

**Comment by nijel** [comment]
> It is not possible to change the interface language in the profile.

Check cookies in your browser (particularly the `django_language` cookie). It works fine for me on https://weblate.eso-spolszczenie.eu/

> Selecting "component list" as the default dasboard view does not work.

It's "Component lists" and it works for me as intended:

![obrazek](https://user-images.githubusercontent.com/212189/111067361-2c806300-84c4-11eb-98c6-d8c9a7aa50d8.png)

**Comment by tomkolp** [comment]
I deleted my cookies and it didn't work. In the dijango panel, I turned on showing the list that I had previously selected. When I turned on this list, everything was back to normal. So this problem arises as soon as the user selects a specific component list and it is disabled in the dijango panel.
I would like to display all the component lists in the dashboard, unfortunately it still has a significant impact on the loading speed. When I select a specific list as my default view, only that list should be loaded. The rest of the lists can only be loaded after you have selected them.
